wordpress-mobile/WordPress-iOS
# Added 1st run tooltip for the updated format bar

Fixes https://github.com/wordpress-mobile/WordPress-Editor-iOS/issues/676

This PR adds a tooltip that notes how the format bar can scroll horizontally. The tip will only display on iPhones in portrait orientation and once the tip is displayed, it will not be displayed again.

**How to test:**
* Run the app from as a fresh install on an iPhone, does the tooltip show in portrait mode when editing?
* Run the app from as a fresh install on an iPhone, does the tooltip not show in landscape mode? Exit edit mode, rotate to portrait mode, re-enter edit mode...does the tooltip appear?
* Run the app from as a fresh install on an iPad, you should not see the tooltip...ever.

**Other things to consider:**
* IME Keyboards (e.g. Japanese - Kana)
* if the user enters edit mode directly from the post list, does the tooltip work?
* if the user enters edit mode from view mode on a post, does the tooltip work?
* Works on iOS7 and iOS8
* Minimize (and maximize) the QuickType bar on the keyboard, does the tooltip display properly in either case?
* Does the tooltip appear when an external keyboard is used and the format bar is at the bottom of the screen?
* If edit mode is dismissed while the tooltip is displayed, does it disappear gracefully and immediately?

![scaled screen shot 2015-07-07 at 9 12 41 pm](https://cloud.githubusercontent.com/assets/154014/8562097/9bf51b4a-24f3-11e5-969f-7856b40c83fe.png)
![scaled screen shot 2015-07-07 at 9 13 50 pm](https://cloud.githubusercontent.com/assets/154014/8562096/9bf3e07c-24f3-11e5-9774-f5f0edd1019f.png)

@diegoreymendez  if you have a moment, can you take a look at this?
